video/livestream-health-exports/proto/exportstream.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto2";
+
+package exportstream.livehealth;
```

Comment:
The package dictates the name of the Ruby gem. I found that if you load two gems of the same names it seems to merge the classes, so you could access:
```
irb(main):017:0> Exportstream::AudioTrack
=> Exportstream::AudioTrack
irb(main):018:0> Exportstream::ExternalVideoViewRecord
=> Exportstream::ExternalVideoViewRecord
```
but I thought `Exportstream::Livehealth::AudioTrack` might be preferable. I'm happy to leave this up to discretion though.

https://developers.google.com/protocol-buffers/docs/reference/ruby-generated

Reply:
An alternative could be:
```
package livehealthexportstream
```
->
```
Livehealthexportstream::AudioTrack
```